documentation/wiki/MSBuild-Environment-Variables.md

```diff
@@ -1,12 +1,32 @@
-# MSBuild environment variables
+# MSBuild environment variables list
 
-- [MsBuildSkipEagerWildCardEvaluationRegexes](#msbuildskipeagerwildcardevaluationregexes)
+This document describes the environment variables that are respected in MSBuild, its purpose and usage. 
+Some of the env variables listed here are unsupported, meaning there is no guarantee that variable will be respected in upcoming release, so please use at your own riks.
```

Comment:
```suggestion
Some of the env variables listed here are unsupported, meaning there is no guarantee that variable or a specific combination of multiple variables will be respected in upcoming release, so please use at your own risk.
```

Possibly we might want to explicitly mention that combining varibles might be risky as well

Reply:
Updated thank you